README.md.gotemplate

```diff
@@ -31,7 +31,7 @@ There are no inputs defined in this step
 | Key | Description | Flags | Default |
 | --- | --- | --- | --- |
 {{- range $input := .Step.Inputs }}
-{{- $description := or (index $input "opts" "description") "" }}
+{{- $description := or (index $input "opts" "description") (index $input "opts" "summary") }}
```

Comment:
i would prefer the summary, that is required, while description is optional based on our latest guide

Reply:
also do not forget to update the Step output part of the template too